content/en/integrations/faq/azure-status-metric.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Azure Status Metric
+kind: faq
+aliases:
+  - /integrations/faq/azure-vm-status-is-not-reporting
+---
+
+## Overview
+
+Datadog provides an `azure.*.status` metric for each [Azure integration][1]. For example, Azure Virtual Machines monitored by Datadog should report `azure.vm.status`, which identifies if the machine is currently running.
```

Comment:
I think it would be better to say:

"Datadog provides an `azure.*.status` metric for each resource monitored via the [Azure integration][1]."

Not sure if that's the right wording exactly- but the idea is that the status metric is provided for the resource, not the integration.